src/docs/src/documentation/content/xdocs/zookeeperAdmin.xml

```diff
@@ -1155,6 +1155,30 @@ server.3=zoo3:2888:3888</programlisting>
             </listitem>
           </varlistentry>
 
+          <varlistentry>
+            <term>4lw.commands.whitelist</term>
+
+            <listitem>
+              <para>(Java system property: <emphasis
```

Comment:
>>This new configuration option is provided as both zoo.cfg option and system properties so users can encode the white list in zoo.cfg and that is the recommended approach as documented in the admin manual

Do you meant, you are supporting both options - users can either configure the list in `zoo.cfg` or set as `system properties`? If yes, I'm OK to this approach.  But in the code I could see that server reads the value from `System.getProperty(ZOOKEEPER_4LW_COMMANDS_WHITELIST)` and it is not reading the value from `zoo.cfg`